osf/migrations/0019_cedarmetadatarecord_update.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.17 on 2024-01-22 14:11
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0018_cedarmetadatarecord'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='cedarmetadatarecord',
+            old_name='guid',
+            new_name='target',
+        ),
+        migrations.AlterUniqueTogether(
+            name='cedarmetadatarecord',
+            unique_together={('target', 'template')},
+        ),
+    ]
```

Comment:
I am okay with renaming the field on the model. However model fields and serializer fields doesn't have to have the same name. It could be `guid` on the model and still have `target` in the serialized API document. The API contract specifies what fields the serialized API document should have, but it has no say on what the actual database table/django models should look like.

Reply:
hmm, I ran into problems when the field name and model name are different. I also checked several other exiting models and serializers, which all have the same name. I can take another look.

Reply:
Ah, there is a `source` attribute that I can use to specified the model field name, which make sense